spec/unit/outputs/elasticsearch/http_client_spec.rb

```diff
@@ -266,6 +266,22 @@
             end
           end
         end
+
+        context "with partial unsupported action" do
```

Comment:
testing scenarios:
 - several valid actions with one trailing invalid action
 - one leading invalid action followed by several valid actions
 - batch of several invalid actions and no valid actions
 - batch of intermixed valid and invalid actions
 - batch of exactly one action that is invalid

Reply:
unit tests cover all these mentioned cases and integration tests guarantee unsupported actions did reach ES with CRUD operation.